experimental/dds/tree2/src/test/shared-tree/sharedTree.spec.ts

```diff
@@ -799,6 +799,69 @@ describe("SharedTree", () => {
 			unsubscribe2();
 		});
 
+		/**
+		 * the collab window includes all sequenced edits after the minimum sequence number
+		 * these tests test that undoing edits behind (i.e., with a seq# < to) the minimum sequence number works
+		 */
+		it("out of collab window", () => {
+			const provider = new TestTreeProviderLite(2);
+			const content = {
+				schema: stringSequenceRootSchema,
+				allowedSchemaModifications: AllowedUpdateType.None,
+				initialTree: ["A", "B", "C", "D"],
+			} satisfies InitializeAndSchematizeConfiguration;
+			const tree1 = provider.trees[0].schematizeInternal(content);
+
+			const { undoStack, redoStack, unsubscribe } = createTestUndoRedoStacks(
+				tree1.checkout.events,
+			);
+
+			provider.processMessages();
+			const tree2 = provider.trees[1].schematizeInternal(content);
+
+			const root1 = tree1.editableTree;
+			const root2 = tree2.editableTree;
+
+			// insert in first tree
+			root1.removeAt(0);
+
+			provider.processMessages();
+			const minimumSequenceNumber = provider.minimumSequenceNumber;
+			assert.deepEqual(root1.asArray, ["B", "C", "D"]);
+			assert.deepEqual(root2.asArray, ["B", "C", "D"]);
+
+			// some more inserts and deletes on the second tree
+			for (let i = 0; i < 5; i++) {
+				root2.insertAt(3, ["y"]);
+				root2.removeAt(3);
+			}
+			provider.processMessages();
```

Comment:
Do we still need this?

Reply:
looks like at least 2 sets of them are required between each tree